src/components/main-menu/MainMenu.test.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import cases from "jest-in-case";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+import MainMenu from "./MainMenu";
+
+test("should display title", () => {
+  render(<MainMenu />);
+
+  expect(screen.getByText(/atom/i)).toBeInTheDocument();
+});
+
+test("should display logo", () => {
+  const { container } = render(<MainMenu />);
+
+  expect(container.querySelector("svg")).toBeInTheDocument();
```

Comment:
query selector was the only option for the current markup.  I'd consider adding a test-id if we want to improve it, but I'm hesitant to change the production code too much without tests in place.

Reply:
It may also require editing the SVG transform - though I'm pretty sure it is receiving all the props passed.